inspect/config.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Configuration</title>
+  <script src="https://docs.getgrist.com/grist-plugin-api.js"></script>
+  <style>
```

Comment:
I'd prefer the inspect example to be as bare-bones as possible, with only essential styling. While it is useful from time to time for inspection, its main purpose is as a functional example to point people to. I don't want to see this css spreading everywhere via cut and paste, making our lives difficult down the line.